modules/geo-layers/src/tile-3d-layer/tile-3d-layer.ts

```diff
@@ -241,7 +241,7 @@ export default class Tile3DLayer<DataT = any, ExtraPropsT extends {} = {}> exten
       return;
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    //@ts-expect-error Type 'Vector3' is not assignable to type '[number, number, number]'
```

Comment:
Got this error and didnt want to go down the rabbit hole of enforcing only `[number, number, number]`. What do you advise? @felixpalmer

Reply:
Furthermore, there was a type mismatch error between the viewport defined here and a viewport type defined in `@loaders.gl`, since the `project`/`unproject` method signatures are changing from `number[]` to `[number, number, number]`. Seems like a PR needs to be made in the `@loaders.gl` repo as well.

Reply:
I believe to have broken something cause now the test hangs on the `Tile3DLayer` initial load step and I do not get an error message...

Reply:
Opened https://github.com/visgl/loaders.gl/pull/3178 to address this.